pytest.ini

```diff
@@ -25,4 +25,5 @@
 [pytest]
 addopts = --pep8 --ignore=docs --cov=invenio_workflows --cov-report=term-missing
 pep8ignore =
+    *.py E501
```

Comment:
can you enable it only for specific files you know that you are going to fix later?